3.8/aql/examples-grouping.md

```diff
@@ -327,3 +327,21 @@ FOR u IN users
 
 To increase readability, the repeated expression *LENGTH(group)* was put into a variable
 *numUsers*. The `FILTER` on *numUsers* is the equivalent an SQL *HAVING* clause.
+
+
+Aggregating data in local time
+-----------------------------
+If you store datetimes in UTC in your collections and need to group data for each day in your local timezone:
+
+```js
```

Comment:
Can you make this an interactively executed example? This one shows how its done:
https://raw.githubusercontent.com/arangodb/docs/main/3.8/aql/examples-combining-graph-traversals.md

You could add the data to be `activities` via a bind variable - the ` @BV {` section in there. 
so change the query to start with `FOR a IN @activities` and have 
```
BV {
      activities: [ { first: doc}, ... 
```

Reply:
Wrote it down blindly.
may need adjustments / Dataset is ignored?

use case: even though data comes from 2 different days UTC they are grouped together in local time.

Reply:
If you provide the list of dates to group using collect for this example you don't depend on collection data. I'd say 5 dates should be enough to demonstrate the effect?

Reply:
Adapted the showcase with inline data and added daylight saving sample

Cannot test it locally with docker unfortunatly:
```
 Liquid Exception: undefined method `each' for "../3.5/generated/arangobackup-options.json":String in 3.5/programs-arangobackup-options.md
bundler: failed to load command: jekyll (/root/gems/bin/jekyll)
```

Reply:
if the example works for you in the browser like this, it should be good. 
You don't need `@DATASET{routeplanner}` since your example works without a dataset. This would [provision the routeplanner example](https://www.arangodb.com/docs/stable/graphs.html#the-city-graph) before executing the AQL query.

Reply:
Adapted it - however I think someone needs to generate the aql example and check it in? It's seems its not generated within the build. And how could it if the other PR is not merged yet.

Reply:
It's a manual process which is somewhat complicated. I added a few more examples, generated and added the generated examples. Can you take a look if they are somewhat meaningful @coronabytes?

Reply:
Yeah somewhat meaningful^^
Maybe we should write something about that these functions reflect historical daylight saving times
like the abolition in China in 1991
and if you don't want that you'll have to use "Etc/GMT-2" for european summer time

and for america that there is no "pacific time" you need to use "America/Los_Angles"
wich reflects PST/PDT